modules/nf-core/kma/index/main.nf

```diff
@@ -0,0 +1,49 @@
+process KMA_INDEX {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/kma:1.4.15--he4a0461_0' :
+        'biocontainers/kma:1.4.15--he4a0461_0' }"
+
+    input:
+    tuple val(meta), path(fasta)
+
+    output:
+    tuple val(meta), path("${meta.id}.kmaindex.*"),   emit: db
```

Comment:
Please make the outputname flexible :)
```suggestion
    tuple val(meta), path("*.kmaindex.*"),   emit: db
```
even better if a custom name would be possible, but sometimes this is difficult with db